tls/s2n_early_data.h

```diff
@@ -58,6 +58,7 @@ S2N_RESULT s2n_early_data_validate_recv(struct s2n_connection *conn);
 
 int s2n_config_set_server_max_early_data_size(struct s2n_config *config, uint32_t max_early_data_size);
 int s2n_connection_set_server_max_early_data_size(struct s2n_connection *conn, uint32_t max_early_data_size);
+int s2n_connection_set_server_early_data_context(struct s2n_connection *conn, const uint8_t *context, uint16_t context_size);
```

Comment:
Should we use `uint32_t` for the context size instead of `uint16_t`?

Reply:
No. The context has to fit in the ticket, and the ticket is limited to uint16_t.